docs/cql/ddl.rst

```diff
@@ -927,6 +937,15 @@ The ``ALTER TABLE`` statement can:
 .. warning:: Once a column is dropped, it is allowed to re-add a column with the same name as the dropped one
    **unless** the type of the dropped column was a (non-frozen) column (due to an internal technical limitation).
 
+Limiting the rate of requests per partition
```

Comment:
Good suggestion, thanks.

Reply:
I've added an anchor to the first occurrence of the paragraph to use in the link instead of the URL (to limit the number of URLs we'll have to update later on).